static/src/js/components/EarthdataDownload/EarthdataDownload.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react'
+import { FaDownload } from 'react-icons/fa'
+import { capitalize } from 'lodash'
+import Button from '../Button/Button'
+
+import eddLogo from '../../../assets/images/earthdataDownload-screenshot.png'
+import unavailableImg from '../../../assets/images/image-unavailable.svg'
+import { getOperatingSystem } from '../../util/files/parseUserAgent'
+
+import { getApplicationConfig } from '../../../../../sharedUtils/config'
+
+import './EarthdataDownload.scss'
+
+export const EarthdataDownload = () => {
+  const { userAgent } = navigator
+  let operatingSystem = getOperatingSystem(userAgent)
+  let downloadLink
+
+  let executableSize
+  const windowsDownloadLink = '//github.com/nasa/earthdata-download/releases/latest/download/Earthdata-Download-x64.exe'
+  const macDownloadLink = '//github.com/nasa/earthdata-download/releases/latest/download/Earthdata-Download-x64.dmg'
+  const macSiliconDownloadLink = '//github.com/nasa/earthdata-download/releases/latest/download/Earthdata-Download-arm64.dmg'
+  const linuxDownloadLink = '//github.com/nasa/earthdata-download/releases/latest/download/Earthdata.Download-amd64.deb'
+  const {
+    macOsEDDExecutableSize,
+    windowsEDDExecutableSize,
+    linuxEDDExecutableSize
+  } = getApplicationConfig()
+
+  let isMacOs = false
```

Comment:
It might make sense to change all these `isMacOs` to `isMacOS` in this file since the `OS` is an abreviation  I don't think we'd want to camelCase it.

Reply:
Agreed. I forgot that the 's' is also capitalized here so I'll update that throughout. Thank you!